resotolib/resotolib/config.py

```diff
@@ -125,7 +125,10 @@ def load_config(self, reload: bool = False) -> None:
                 for config_id, config_data in config.items():
                     if config_id in Config.running_config.classes:
                         log.debug(f"Loading config section {config_id}")
-                        new_config[config_id] = jsons.load(config_data, Config.running_config.classes[config_id])
+                        clazz: type = Config.running_config.classes.get(config_id, Any)
+                        # use the from_json class from config, if available
+                        loader = getattr(clazz, "from_json", jsons.load)
+                        new_config[config_id] = loader(config_data)
```

Comment:
Doesn't jsons.load require a second argument that is the class name you want the json data to marshal to?

Reply:
absolutely right. fixed.